src/ucs/debug/debug.c

```diff
@@ -881,10 +882,8 @@ void ucs_handle_error()
 static void ucs_debug_signal_handler(int signo)
 {
     ucs_log_flush();
-
-    ucs_log_fatal_error("Got debug signal, raising log level",
-                        ucs_get_host_name(), getpid());
```

Comment:
this is removed because now the debug signal also dumps profile data. also when sending this signal (with pdsh) to a large job it would fill the screen with output (message from each process) so i used this opportunity to remove it.